release_notes/v0.8.md

```diff
@@ -0,0 +1,7 @@
+# What’s New in OPEA v0.8
+- Broaden functionality coverage
+    - OPEA v0.8 Support LVM(Large Vision Models), it provides a microservice for llava on Intel Gaudi2. It also create an initial Agent microservice, OPEA now has the capability to process information, evaluate situations, and take actionable steps. Meanwhile more components are coming, such as vLLM on Ray, giving users more choices for different AI functionalities. FAQGen (Frequently Asked Questions Generation) is a new example of LLM application.
+- Deploy across more platforms
+    - We validate ChatQnA on AWS, including GenAI Microservices Connector (GMC), Kubernetes manifests and docker. Meanwhile it can work on AIPC. Kubernetes manifests for ChatQnA, DocSum, CodeGen and CodeTrans are created. The document for deploying OPEA with Docker Hub is also included in this release. GMC supports the launching, monitoring, and updating of GenAI microservice Applications on Kubernetes.
```

Comment:
s/validate/validated
s/AWS/AWS and Nvidia H100

Add one more sentence ..
This release also supports basic Authentication and Authorization using JWT tokens.

Reply:
not sure how to add AWS and Nvidia H100, current sentence is "OPEA pipeline can also operate on AIPC and Nvidia H100"
one more sentence has been added